samples/estimation/df-chemistry/chemistry.py

```diff
@@ -10,6 +10,7 @@
 from scipy import linalg as LA
 from urllib.parse import urlparse
 from urllib.request import urlretrieve
+from qsharp_widgets import EstimatesPanel
```

Comment:
It seems that this is not used in this change. Would it be possible to use the widgets instead of manually creating the table in text? Otherwise, maybe a pandas DataFrame could be the better option.

Reply:
Widgets were not a good choice for commandline. I can try DataFrame and remove this import statement.

Reply:
@msoeken I have added data using dataframes but it does not have the grid for better visuals. In case it is ok to add one more library tabulate, the visuals would be as follows:
<img width="509" alt="image" src="https://github.com/microsoft/qsharp/assets/10944026/e8d08e9d-9ac3-4584-b2af-903b9be8c814">

Please let me know if you find this one better.

Reply:
It's not a very common library. How about the markdown output for DataFrames: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_markdown.html

Reply:
Sure, I have updated the code.

Reply:
When I try to run this code, it requires pandas (expected) and then tabulate. This conversation sounded like we don't want to add tabulate as a dependency; what's the final verdict on this?

Reply:
@devikamehra Have you seen the discussion about the tabulate dependency that has crept into the PR?

Reply:
@tcNickolas No Mariia, Thank you driving my attention on this. Will verify if tabulate required for installation because I did not include it as per the discussion with Mathias.

Reply:
@msoeken @tcNickolas Got it, I have updated the code such that we do not require pandas at all. Let me know if this plain vanilla code is ok or not.